platform/ui-next/src/components/DataRow/DataRow.tsx

```diff
@@ -97,6 +98,7 @@ export const DataRow: React.FC<DataRowProps> = ({
   isVisible = true,
   disableEditing = false,
   className,
+  isUnmapped = false,
```

Comment:
I'm not sure about this, we should not mention measurement service logic (mapped vs unmapped) in the ui. Maybe have it as a slot?